### ISSUE TITLE
NTR - protein targeting to lysosome involved in chaperone-mediated autophagy

### ISSUE THREAD

**Issue by Pauldenny** [geneontology/go-ontology#12109]
GO:0061684 chaperone-mediated autophagy (CMA) currently has only regulation children, but is a relatively well-understood process
Wikipedia:
https://en.wikipedia.org/wiki/Chaperone-mediated_autophagy
The first step in CMA is detection / recognition of substrate proteins, by KFERQ-motif related sequence binding to hsc70 (humans Uniprot P11142).  This complex, also with (uncertain) co-chaperone proteins, is then _somehow_ targeted to lysosome.
I would like to have a new term, similar to: 
protein targeting to lysosome involved in chaperone-mediated autophagy
This term would define a CMA sub-process and would be used to annotate hsc70-related proteins plus co-chaperones, as they are identified e.g. in PMID_26212789, PMID_18644871, etc.
Parents would probably be:
GO:0061684 chaperone-mediated autophagy 
GO:0006622	protein targeting to lysosome  
It would also be very useful to have the related terms:

protein unfolding involved in chaperone-mediated autophagy
(important part of preparing substrate for transport across lysosomal membrane) 
protein import into lysosome
protein import into lysosome involved in chaperone-mediated autophagy

**Comment by ukemi** [comment]
Hi Paul,

How about something like this?

CMA
intersection of starts_with chaperone binding
intersection of has_part receptor binding
intersection of has_part chaperone mediated protein transport
intersection of ends_with protein catabolic process

New terms: 
protein targeting to lysosome involved in chaperone-mediated autophagy
intersection of is_a protein targeting to lysosome
intersection of part_of CMA
has_part chaperone binding
has_part receptor binding
is_a protein targeting to lysosome
part_of CMA

chaperone mediated protein transport involved in CMA
intersection of chaperone mediated protein transport
intersection of has_target_end_location lysosome
intersection of part_of CMA
is_a chaperone mediated protein transport
has_part protein unfoding
part_of CMA
part_of CMA

protein catabolic process involved in CMA
intersection of protein catablic process
intersection of part_of CMA
is_a protein catablic process
part_of CMA

**Comment by Pauldenny** [comment]
Looks good - thanks David!

**Comment by ukemi** [comment]
Hi Paul,

Here is what I added/changed:

name: chaperone-mediated autophagy
 namespace: biological_process
-def: "The autophagy process which begins when chaperones and co-chaperones recognize a KFERQ-related motif and unfold the substrate protein. The process ends with the translocation of the protein across the lysosomal membrane." [GOC:pad, GOC:PARL, PMID:22743996]
+def: "The autophagy process which begins when chaperones and co-chaperones recognize a KFERQ-related motif and unfold the substrate protein. The process ends with the catabolism of the protein in the lysosome." [GOC:pad, GOC:PARL, PMID:22743996]
 synonym: "CMA" RELATED [PMID:22743996]
 is_a: GO:0006914 ! autophagy
-is_a: GO:0019538 {is_inferred="true"} ! protein metabolic process
-relationship: ends_with GO:0061462 ! protein localization to lysosome
-relationship: has_part GO:0006622 ! protein targeting to lysosome
-relationship: starts_with GO:0043335 ! protein unfolding
+intersection_of: GO:0006914 ! autophagy
+intersection_of: ends_with GO:0030163 ! protein catabolic process
+intersection_of: has_part GO:0005102 ! receptor binding
+intersection_of: has_part GO:0072321 ! chaperone-mediated protein transport
+intersection_of: starts_with GO:0051087 ! chaperone binding
 created_by: dph
 creation_date: 2015-03-04T14:35:26Z
 
@@ -423203,6 +423204,33 @@
 creation_date: 2015-10-14T12:02:01Z
 
 [Term]
+id: GO:0061740
+name: protein targeting to lysosome involved in chaperone-mediated autophagy
+namespace: biological_process
+def: "The targeting of a protein to the lysosome process in which an input protein binds to a chaperone and subsequently to a lysosomal receptor." [GOC:dph, GOC:pad, GOC:PARL, PMID:22748206]
+is_a: GO:0006622 ! protein targeting to lysosome
+intersection_of: GO:0006622 ! protein targeting to lysosome
+intersection_of: part_of GO:0061684 ! chaperone-mediated autophagy
+relationship: has_part GO:0005102 ! receptor binding
+relationship: has_part GO:0051087 ! chaperone binding
+relationship: part_of GO:0061684 ! chaperone-mediated autophagy
+created_by: dph
+creation_date: 2015-10-21T08:41:29Z
+
+[Term]
+id: GO:0061741
+name: chaperone-mediated protein transport involved in chaperone-mediated autophagy
+namespace: biological_process
+def: "The chaperone-mediated protein transport process in which a protein that is bound to a chaperone and a lysosomal receptor is unfolded and transported into the lysosome as part of chaperone-mediated autophagy." [GOC:autophagy, GOC:dph, GOC:pad, GOC:PARL, PMID:22748206]
+is_a: GO:0072321 ! chaperone-mediated protein transport
+intersection_of: GO:0072321 ! chaperone-mediated protein transport
+intersection_of: part_of GO:0061684 ! chaperone-mediated autophagy
+relationship: has_part GO:0043335 ! protein unfolding
+relationship: part_of GO:0061684 ! chaperone-mediated autophagy
+created_by: dph
+creation_date: 2015-10-21T08:56:33Z

I decided not to add the specific protein catabolism term because once the protein is in the lysosome, it seems that the process would be the same as any other protein being degraded in the lysosome. Let me know if you disagree.

-D
